src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonerSearchPrisoner.kt

```diff
@@ -4,4 +4,8 @@ data class PrisonerSearchPrisoner(
   val prisonerNumber: String,
   val bookingId: String,
   val status: String? = null,
+  val licenceExpiryDate: String? = null,
```

Comment:
You can push the responsibility of deserialization into jackson by making these `LocalDate`

This principle of pushing type checking to the boundaries of an application is really nice and you can see the effect of not doing this in the frontend - date parsing everywhere!

Might need to add `@JsonFormat(pattern = "yyyy-MM-dd")`

Reply:
Done